internal/common/gamestate/gamestate_test.go

```diff
@@ -39,8 +39,12 @@ func TestGetClientGameStateCopy(t *testing.T) {
 				CriticalConsecutiveTurnsCounter: 2,
 			},
 		},
-		IIGORolesBudget: map[shared.Role]shared.Resources{},
-		CommonPool:      20,
+		IIGORolesBudget: map[shared.Role]shared.Resources{
+			shared.Judge:     shared.Resources(10),
```

Comment:
Don't need the cast, but doesn't really matter it's there

Reply:
I don't know how did that happen because I didn't touch this file